python/gstaichi/lang/_fast_caching/args_hasher.py

```diff
@@ -125,7 +125,7 @@ def hash_args(args: Sequence[Any], arg_metas: Sequence[ArgMetadata | None]) -> s
         start = time.time()
         _hash = stringify_obj_type((str(i_arg),), arg, arg_metas[i_arg])
         g_repr_time += time.time() - start
-        if not _hash:
+        if _hash is None:
```

Comment:
`not _hash` also includes emptyt string ,which was causing empty data oriented to appear as invalid arg.

Reply:
Updated to no longer allow empty hash as valid hash.